clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -253,3 +253,16 @@ suspend fun FossIdRestService.listIgnoredFiles(user: String, apiKey: String, sca
             "get_ignored_files", SCAN_GROUP, user, apiKey, "scan_code" to scanCode
         )
     )
+
+/**
+ * List the files of a scan that are in "pending identification" state
+ * @param user the user querying the API
+ * @param apiKey the key to query the API
+ * @param scanCode the code of the scan
```

Comment:
Side note: This is fine for now as it aligns with the existing code in this file. But at some point we should change all function docs to use the preferred KDoc style of parameter references inlined into fluent text.

Reply:
Ok. When this one in merged I will do a PR for it.